src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -34,6 +34,10 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".frm")>
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".pag")>
     <ProvideLanguageExtension(GetType(VisualBasicLanguageService), ".vb")>
+    <ProvideLanguageCodeExpansion(GetType(VisualBasicLanguageService), "Basic", 101, "VB", "%InstallRoot%\VB\Snippets\%LCID%\SnippetIndex.xml",
```

Comment:
Does the value 101 for "language name identifier" have to be defined somewhere? similar to how 115 for the csharp has an entry in the resx?